frontend/src/container/LiveLogs/LiveLogsList/index.tsx

```diff
@@ -0,0 +1,139 @@
+import { Card, Typography } from 'antd';
+import ListLogView from 'components/Logs/ListLogView';
+import RawLogView from 'components/Logs/RawLogView';
+import Spinner from 'components/Spinner';
+import { LOCALSTORAGE } from 'constants/localStorage';
+import InfinityTableView from 'container/LogsExplorerList/InfinityTableView';
+import { InfinityWrapperStyled } from 'container/LogsExplorerList/styles';
+import { convertKeysToColumnFields } from 'container/LogsExplorerList/utils';
+import { Heading } from 'container/LogsTable/styles';
+import { useOptionsMenu } from 'container/OptionsMenu';
+import { contentStyle } from 'container/Trace/Search/config';
+import { useCopyLogLink } from 'hooks/logs/useCopyLogLink';
+import useFontFaceObserver from 'hooks/useFontObserver';
+import { useEventSource } from 'providers/EventSource';
+import { memo, useCallback, useEffect, useMemo, useRef } from 'react';
+import { Virtuoso, VirtuosoHandle } from 'react-virtuoso';
+// interfaces
+import { ILog } from 'types/api/logs/log';
+import { DataSource, StringOperators } from 'types/common/queryBuilder';
+
+import { LiveLogsListProps } from './types';
+
+function LiveLogsList({ logs }: LiveLogsListProps): JSX.Element {
+	const ref = useRef<VirtuosoHandle>(null);
+
+	const { isConnectionError, isConnectionLoading } = useEventSource();
+
+	const { activeLogId } = useCopyLogLink();
+
+	const { options } = useOptionsMenu({
+		storageKey: LOCALSTORAGE.LOGS_LIST_OPTIONS,
+		dataSource: DataSource.LOGS,
+		aggregateOperator: StringOperators.NOOP,
+	});
+
+	const activeLogIndex = useMemo(
+		() => logs.findIndex(({ id }) => id === activeLogId),
+		[logs, activeLogId],
+	);
+
+	useFontFaceObserver(
+		[
+			{
+				family: 'Fira Code',
+				weight: '300',
+			},
+		],
+		options.format === 'raw',
+		{
+			timeout: 5000,
+		},
+	);
+
+	const selectedFields = useMemo(
```

Comment:
useMemo isn't required here.

https://blog.logrocket.com/when-not-to-use-usememo-react-hook/#:~:text=It's%20important%20to%20note%20that,itself%20has%20a%20small%20overhead.